test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -411,6 +411,29 @@ def printSummary(self):
         self.logger.info('\n'.join(msg))
 
 
+class SystemInfo:
+    """
+        used to get system's ram size and disk size information.
+
+        :system_ram: ram of the test running machine
+        :system_disk: disk size of the test running machine
+    """
+    def __init__(self, hostname=None):
+        du = DshUtils()
+        # getting RAM size in gb
```

Comment:
Here i mentioned the unit

Reply:
It should be in the docstring, else it won't be documented.